cmd/agent/gui/views/private/css/stylesheet.css

```diff
@@ -324,6 +324,7 @@ a {
   margin: 0;
   padding: 0 0 0 20px;
 }
+
```

Comment:
Lets revert this (whitespace only) change, and the same in `.gitignore`. 
To easily revert this to what is in `main` you can use: 

`git checkout main -- cmd/agent/gui/views/private/css/stylesheet.css`
`git checkout main -- .gitignore`

These commands will check the file out from the other branch - effectively reverting them back to exactly how they were.